src/Gtransfo.cc

```diff
@@ -1573,11 +1561,10 @@ GtransfoLin BaseTanWcs::LinPart() const
 
 void BaseTanWcs::SetCorrections(const GtransfoPoly *Corrections)
 {
-  if (corr) delete corr;
-  corr = nullptr;
   if (Corrections)
     {
-      corr = (GtransfoPoly*) Corrections->Clone();
+      // Unfortunate construct to cast unique_ptr, requires identical deleter!
```

Comment:
No, something we could cleanup by not casting. But in this case the function was actually not needed anyway, so I could just replace it with a `std::unique_ptr<GtransfoPoly>` argument and push the cloning into the caller (which is non-existent).